### ISSUE TITLE
Simple Stack creation fails for Flex shape 

### ISSUE THREAD

**Issue by aleksey-korolev** [oracle-quickstart/oci-elastic#18]
In the creation wizard I chose instance shape to be VM.Standard.A1.Flex and terraform job failed with following output:

```
oci_core_instance.ELK: Creating...
Error: 400-InvalidParameter
Provider version: 4.61.0, released on 2022-01-26.
Service: Core Instance
Error Message: Invalid ShapeConfig: null (Cannot launch flexible instance without ShapeConfig)
OPC request ID: dfc1fe5ec2ab1acdcd6f9fec24a683fe/E02B897D8F246ADFF2299C8281A0A14C/271139532B7970A1250A888063D98B7F
Suggestion: Please Update the parameter(s) in the Terraform config as per error message Invalid ShapeConfig: null (Cannot launch flexible instance without ShapeConfig)
  on compute.tf line 18, in resource "oci_core_instance" "ELK" 
  18: resource "oci_core_instance" "ELK" {
```

After I changed to VM.Standard1.1 job finished successfully.

**Comment by lfeldman** [comment]
@aleksey-korolev could you please re-test the code with Terraform CLI? I have merged new changes.

**Comment by lfeldman** [comment]
@aleksey-korolev I have just released the [latest 2.0 release](https://github.com/oracle-quickstart/oci-elastic/releases/tag/v2.0). I close the problem of the ARM shapes broadly in all three scenarios.
